### DISCUSSION TITLE
cant upsert

### DISCUSSION THREAD

**Discussion by viuna-ka** [FlowiseAI/Flowise#3461] [Q&A]
![Capture](https://github.com/user-attachments/assets/32323124-60ac-45db-b0b7-82780f8b4ae7)
 i cant upsert this that shown me this 
Oh snap!
The following error occured when loading this page.
Status: 500
Error: documentStoreServices.insertIntoVectorStore - Error: documentStoreServices._insertIntoVectorStoreWorkerThread - TypeError: fetch failed
Please retry after some time. If the issue persists, reach out to us on our Discord server.
Alternatively, you can raise an issue on Github.

**Comment by dickyardi** [comment]
Did you find the solution?
